Aladin-iOS/Aladin-iOS/Global/Extension/addComma.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  addComma.swift
+//  Aladin-iOS
+//
+//  Created by 김인영 on 2022/11/28.
+//
+
+import Foundation
+
+//MARK: - 천단위마다 ,표시
+func addComma(value: Int) -> String{
+    let numberFormatter = NumberFormatter()
+    numberFormatter.numberStyle = .decimal
+    let result = numberFormatter.string(from: NSNumber(value: value))! + "원"
+    
+    return result
+}
```

Comment:
코드 컨벤션 들여쓰기 한번 봐주시고, 요런 방식으로 강제언래핑 사용을 안 할 수도 있어요

```suggestion
func addComma(value: Int) -> String {
    let numberFormatter = NumberFormatter()
    numberFormatter.numberStyle = .decimal
    let result = numberFormatter.string(from: NSNumber(value: value)) ?? ""
    
    return result + "원"
}
```